backend/svirt.pm

```diff
@@ -174,6 +175,30 @@ sub load_snapshot ($self, $args) {
     return $post_load_snapshot_command;
 }
 
+sub do_extract_assets ($self, $args) {
+    my $first_hdd = $bmwqemu::vars{S390_ZKVM} ? 'a' : 'b';
+    my $name = $args->{name};
+    die "No name given" unless $args->{name};
+    $name =~ /\.([[:alnum:]]+)$/;
+    die "No HDD specified" unless $args->{hdd_num};
+    my $hdd_num = $args->{hdd_num} - 1;
+    die "No format specified" unless $args->{format};
+    my $format = $args->{format};
+    return undef if (($format ne 'raw') and ($format ne 'qcow2'));
+    my $svirt = $self->select_console('svirt');
```

Comment:
I think the usage is:
`$self->select_console({testapi_console => 'svirt'});`

Reply:
Maybe `select_console` isn't even needed and just
`my $svirt_name = $self->console('svirt')->name . ...;`

Reply:
Actually I'll use `my $vmname = $self->console('svirt')->name` here like elsewhere, so this is consistent